storage-proofs/porep/src/stacked/vanilla/cache.rs

```diff
@@ -17,6 +18,7 @@ use storage_proofs_core::{
 use super::graph::{StackedGraph, DEGREE};
 
 /// Path in which to store the parents caches.
+pub const PARENT_CACHE_ENV_VAR: &str = "FIL_PROOFS_PARENT_CACHE";
```

Comment:
why not use `settings.rs` for this?

Reply:
Yes, please.

Reply:
I went with the existing approach taken by the PARAMETER_CACHE

Reply:
But can move them both to settings